Bottom: ef12221b9b980010ecf33567429f0b17dee2a194
Top:    4977b9a4f1756d2a337e038303c1261f8da0b398
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 13:41:23 +0100

toporouter.[ch]: Add generic "contributors" copyright notice

"Copyright (C) 2009-2011 PCB Contibrutors (See ChangeLog for details)"


---

diff --git a/src/toporouter.c b/src/toporouter.c
index e5310b0..a89679c 100644
--- a/src/toporouter.c
+++ b/src/toporouter.c
@@ -4,6 +4,7 @@
  *  Topological Autorouter for 
  *  PCB, interactive printed circuit board design
  *  Copyright (C) 2009 Anthony Blake
+ *  Copyright (C) 2009-2011 PCB Contributors (see ChangeLog for details)
  *
  *  This program is free software; you can redistribute it and/or modify
  *  it under the terms of the GNU General Public License as published by
diff --git a/src/toporouter.h b/src/toporouter.h
index fcd0053..b6c48b4 100644
--- a/src/toporouter.h
+++ b/src/toporouter.h
@@ -4,6 +4,7 @@
  *  Topological Autorouter for 
  *  PCB, interactive printed circuit board design
  *  Copyright (C) 2009 Anthony Blake
+ *  Copyright (C) 2009-2011 PCB Contributors (see ChangeLog for details)
  *
  *  This program is free software; you can redistribute it and/or modify
  *  it under the terms of the GNU General Public License as published by
